blog-cse/2022/12-31.md

```diff
@@ -453,7 +453,7 @@ Cloud SIEM now supports custom sources of inventory data. Now, if you want to in
 
 #### Standard Match Lists
 
-As a reminder, the migration for our out-of-the-box rules content from standard match lists to tags for Entities has begun. The system is now automatically setting the appropriate tags for any Entities appearing in any of the standard match lists called out in the [previous announcement](https://help.sumologic.com/release-notes-cse/2022/10/13/application-update/). This will continue until January 20, 2023, when the migration will be complete.
+As a reminder, the migration for our out-of-the-box rules content from standard match lists to tags for Entities has begun. The system is now automatically setting the appropriate tags for any Entities appearing in any of the standard match lists called out in the [previous announcement](/release-notes-cse/2022/10/13/application-update/). This will continue until January 20, 2023, when the migration will be complete.
```

Comment:
```suggestion
As a reminder, the migration for our out-of-the-box rules content from standard match lists to tags for Entities has begun. The system is now automatically setting the appropriate tags for any Entities appearing in any of the standard match lists called out in the [previous announcement](/release-notes-cse/2022/12/31/#october-13-2022---application-update). This will continue until January 20, 2023, when the migration will be complete.
```